### DISCUSSION TITLE
1.4.0

### DISCUSSION THREAD

**Discussion by Mirobit** [Mirobit/bitcoin-node-manager#66] [General]
Have a look at the [Changelog](https://github.com/Mirobit/bitcoin-node-manager/blob/master/CHANGELOG.md) for more information.

<hr /><em>This discussion was created from the release <a href='https://github.com/Mirobit/bitcoin-node-manager/releases/tag/1.4.0'>1.4.0</a>.</em>

**Comment by GALAXYPEHUB** [comment]
How to set a precise amount of data to be downloaded from the blockchain network
